src/config/data/faq.js

```diff
@@ -4,151 +4,98 @@ const liSpace = { marginBottom: "10px" };
 
 const faqs = [
   {
-    title: <>What is a Hackathon?</>,
+    title: <>What is a hackathon?</>,
     description: (
       <>
-        A hackathon is an event where students of all academic backgrounds and
-        skill levels come together to solve problems by using technology.
-        <br /> <br />
-        Students form groups and create projects that tackle these challenges,
-        all while attending workshops to develop new skills, receiving
-        mentorship from industry leaders, and networking with recruiters from
-        top companies.
+        A hackathon is a weekend-long event where students come together to 
+        learn about the latest technologies and build innovative products.
       </>
     )
   },
   {
-    title: <>Is there a theme?</>,
+    title: <>When and where is ShellHacks?</>,
     description: (
       <>
-        Our theme focuses on diversity, and not just from an academic standpoint.
-        Miami is a cultural melting pot full of people from all walks of life,
-        with{" "}
-        <a
-          className="links"
-          rel="noopener noreferrer"
-          target="_blank"
-          href="https://fiu.edu"
-        >
-          FIU being no exception
-        </a>
-        .
+        ShellHacks will take place from September 20th through the 22nd at 

```

Comment:
Need to add a link for Kovens Center

Reply:
http://kovens.fiu.edu

Reply:
I dont see a need to link the kovens center